_rules/meta-refresh-no-delay-bc659a.md

```diff
@@ -118,130 +118,165 @@ Redirects after more than 20 hours.
 
 #### Failed Example 1
 
-Refreshes after 30 seconds.
+This page refreshes after 30 seconds.
 
 ```html
 <head>
 	<meta http-equiv="refresh" content="30" />
 </head>
+<body>
+	<p>This page refreshes after 30 seconds.</p>
+</body>
 ```
 
 #### Failed Example 2
 
-Redirects after 30 seconds.
+The page redirects to a new page after 30 seconds.
 
 ```html
 <head>
 	<meta http-equiv="refresh" content="30; URL='https://w3.org'" />
 </head>
+<body>
+	<p>This page redirects afte 30 seconds.</p>
+</body>
 ```
 
 #### Failed Example 3
 
-First `<meta http-equiv="refresh">` element is not valid, second one redirects after 5 seconds.
+The first `meta` element on the page is not valid because it uses colon (":") rather than semicolon (";"). The second `meta` element redirects to a new page after 5 seconds.
```

Comment:
```suggestion
The first `meta` element on this page is not valid because it uses colon (":") rather than semicolon (";"). The second `meta` element redirects to a new page after 5 seconds.
```